src/components/Molecules/Task/story.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Component from "./index";
+
+export default {
+  component: Component,
+  title: "Molecules/Task",
+  parameters: {
+    backgrounds: {
+      default: "dark",
+    },
+  },
+};
+
+const Template = (args) => <Component {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  onEditComplete: () => console.log("taskname changed ${taskName}"),
+  taskName:"taskname",
+  onTaskcomplete:() => console.log("task completed"),
```

Comment:
```suggestion
  onTaskComplete:() => console.log("task completed"),
```
ここ修正すればlogがちゃんと出そうです。